src/util/updateMaxHeight.tsx

```diff
@@ -17,7 +19,8 @@ export const updateMaxHeight = (
   const below = belowElements
     ? belowElements[0].getBoundingClientRect().height + 1
     : 0;
-  const offset = Math.ceil(above + below + additionalOffset);
+  const statusBarHeight = getAbsoluteHeightBelow("status-bar");
+  const offset = Math.ceil(above + below + statusBarHeight + additionalOffset);
```

Comment:
Here we have the same situation as with the `ScrollableTable` and `ScrollableContainer` components. There are use cases like from `CustomVolumeCreateModal` where we don't need to take the status bar into account.

Reply:
this one is slightly more annoying to adjust, since the arguments to `updateMaxHeight` are all positional, if I add an extra argument `belowIds` then I would have to pass in a bunch of undefined inputs to `updateMaxHeight` everywhere that it is called. I'm thinking maybe I will define an input for considering the status bar or not, then default it to `true`, then only in specific locations I'd have to disable this behaviour for example in `CustomVolumeCreateModal`

Reply:
Yeah, it has a lot of usages. Adding a property is probably the easiest way to get the few exceptions right.

Reply:
I can't seem to find any other relevant places than `CustoVolumeCreateModal`, not sure if I missed some?

Reply:
I think that is right, only that one I could spot as well!